velox/exec/WindowFunction.h

```diff
@@ -111,10 +111,11 @@ class WindowFunction {
       const core::QueryConfig& config);
 
  protected:
-  // Set result NULL for rows with invalid frames in the input.
-  void setNullEmptyFramesResults(
+  // Set result to default value for rows with empty frames.
+  void setEmptyFramesResult(
       const SelectivityVector& validRows,
       vector_size_t resultOffset,
+      const VectorPtr& defaultResult,
```

Comment:
perhaps, clarify that defaultResult is a vector of size 1.

Reply:
Updated.